src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/TypeBuilderImpl.cs

```diff
@@ -63,7 +64,7 @@ internal sealed class TypeBuilderImpl : TypeBuilder
                 for (int i = 0; i < interfaces.Length; i++)
                 {
                     Type @interface = interfaces[i];
-                    // cannot contain null in the interface list
+                    // cannot contain null in the iface list
```

Comment:
nit: not sure why this was changed to iface

Reply:
Renaming in VS updating the comments too, I had reverted those but looks not all covered